### PR TITLE
Added upload button

### PR DESCRIPTION
**Pull request by BHenkemans**
Closes GH-1715

### CODE REVIEW

File: module/Decision/view/decision/decision/view.phtml [hunk 1]
```diff
@@ -31,8 +31,19 @@ $this->headScript()
...
 <section class="section">
     <div class="container">
-        <h1><?= $meetingName ?></h1>
-        <h2><?= $this->dateFormat($meeting->getDate(), IntlDateFormatter::FULL) ?></h2>
+        <div class="row">
+            <div class="col-md-10">
+                <h1><?= $meetingName ?></h1>
+                <h2><?= $this->dateFormat($meeting->getDate(), IntlDateFormatter::FULL) ?></h2>
+            </div>
+            <?php if ($this->acl('decision_service_acl')->isAllowed('meeting','upload_document') && $meeting->getType()->value != "BV"): ?>
```
Comment (tomudding) [inline module/Decision/view/decision/decision/view.phtml:39]: Please use the `MeetingTypes` enum:

```php
MeetingTypes::BV
```

And also strict `===` comparisons

**Review by tomudding — CHANGES_REQUESTED** [review]
One minor change requested, otherwise this looks great!

**Comment by tomudding** [comment]
@BHenkemans when you have time, can you rebase this?

**Comment by BHenkemans** [comment]
Abandoned. Will restart.

**Pull request closed without merging**